web/themes/custom/pineapple/sass/layouts/_hero-image.scss

```diff
@@ -0,0 +1,54 @@
+.hero-container {
```

Comment:
Please read the following resources, they will help you work out the correct places that sass files should go into.

https://www.drupal.org/node/1887922#file-structure
https://www.drupal.org/node/1887918#separate-concerns

Reply:
Done